src/store-api/src/sst_entry.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_time::Timestamp;
+use serde::{Deserialize, Serialize};
+
+use crate::storage::{RegionGroup, RegionId, RegionNumber, RegionSeq, TableId};
+
+/// An entry describing a SST file known by the engine's manifest.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct ManifestSstEntry {
```

Comment:
We also need index size or index metadata.

Reply:
Added index file path and size. Index metadata will be added to a separate table.